.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "seneca"
+  "extends": "seneca",
+  "rules": {
```

Comment:
ahah.
Problem is that `lab` gives error for warning.
And this specific one should be a bit flexible.

So I found it more wise to disable the error, than to fix the ~30 capitalized name.